### ISSUE TITLE
Decision tress classifier

### ISSUE THREAD

**Issue by Hammadwakeel** [Hammadwakeel/Programming-for-Ai#2]
decision tree classifier python code from scratch

#importing important libraries
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd
import sklearn

#loading the dataset
dataset = pd.read_csv('SR_data.csv')

#selecting the features
X = dataset.loc[:, ['F1','F2' ,'F3' ,'F4','F5']].values

#selecting the labels
y = dataset.loc[:,'Lablel'].values

#spliting the dataset into test and train
from sklearn.model_selection import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.20, random_state = 0)

#importing the classifier
from sklearn.tree import DecisionTreeClassifier
model = DecisionTreeClassifier()
model.fit(X_train, y_train)

#testing the dataset
y_pred = model.predict(X_test)

#checking the accuracy and making confusion matrix
from sklearn.metrics import confusion_matrix,accuracy_score
cm = confusion_matrix(y_test, y_pred)
ac = accuracy_score(y_test,y_pred)
cm

ac

dataset= [SR_Data.csv](https://github.com/Hammadwakeel/Programming-for-Ai/files/9844946/SR_Data.csv)
